subt_ign/worlds/simple_cave_01.tsv

```diff
@@ -1,18 +1,18 @@
-																Cave Corner 01,180,2	Cave Straight 02,90,2	Cave Cavern Split 02,0,2,Rescue Randy@3.29 8.14 -0.5 0 0 15	Cave Cavern Split 01,0,2,Cave Cap@0 37.5 25 0 0 180;Cave Cap@25 0 0 0 0 90			
-																Cave Elevation,180,0						
-															Cave Corner 01,180,0	Cave Corner 02,0,0						
-															Cave Straight 01,180,0							
-														Cave Corner 01,180,0,Rescue Randy@-2 3.14 -0.29 -6.3 0 18	Cave 3 Way 01,0,0	Cave Elevation,-90,-2	Cave Straight 03,90,-2,Phone@0 1.72 0.19 -90 0 20	Cave Elevation,-90,-4	Cave Elevation,-90,-6	Cave Corner 01,90,-6		
-							Cave Corner 02,180,0	Cave Straight 03,90,0	Cave Straight 02,90,0,Backpack@8.83 -1.25 0.31 0 0 138	Cave Straight 01,-90,0	Cave Straight 02,-90,0	Cave Corner 01,90,0		Cave Straight 01,180,0						Cave Straight 01,180,-6,Phone@0 0.64 0.39 -86.66 0 0		
-							Cave Straight 02,0,0					Cave Corner 02,-90,0,Backpack@0 7.58 0.06 0 0 190	Cave Straight 03,-90,0	Cave Corner 02,0,0						Cave Corner 01,-90,-6	Cave Straight 02,-90,-6,Rescue Randy@11.6 0.05 -0.02 0 0 -90	Cave Cap,90,-6
-base_station,0,0	Cave Straight 05,90,0	Cave Straight 01,90,0	Cave Straight 03,90,0,Backpack@5.16 -5 0.07 0 0 180	Cave 3 Way 01,180,0	Cave Straight 02,-90,0	Cave Straight 01,90,0,Phone@-0.04 -2.03 0.41 -95.11 0 83	Cave Corner 01,0,0															
-				Cave Straight 03,0,0																		
-				Cave Straight 01,180,0																		
-				Cave Straight 03,180,0																		
-				Cave Corner 02,-90,0	Cave Corner 01,90,0																	
-					Cave Straight 05,0,0,Phone@0 3.32 0.21 -93.96 0 0																	
-				Cave Corner 01,180,0	Cave 3 Way 01,0,0	Cave Corner 02,90,0																
-				Cave Straight 04,0,0		Cave Vertical Shaft,180,0																
-				Cave Corner 02,-90,0	Cave Vertical Shaft,-90,0,Backpack@5.11 0 0.03 0 0 -80	Cave 3 Way 01,90,4																
-						Cave Straight 02,0,4,Rescue Randy@0 -12.28 0 0 0 185																
-						Cave Cap,0,4																
+																Cave Corner 01,180,2	Cave Straight 02,90,2	Cave Cavern Split 02,0,2,Rescue Randy@3.29 8.14 -0.5 0 0 15	Cave Cavern Split 01,0,2 Type B,Cave Cap@0 37.5 25 0 0 180;Cave Cap@25 0 0 0 0 90			
```

Comment:
The `Type B` string is misplaced. It should come after the cave model name. I think this applies to all 3 simple cave tsv files.

Reply:
Fixed in 86fa520e89cff2e6b63fe6ae4c3461eedfa23823